themes/custom/droopler_theme/scss/components/_d_lang-dropdown.scss

```diff
@@ -107,7 +108,6 @@ $secondary-menu-lang-dropdown-height: 2.4rem !default;
         }
         a {
           font-size: $lang-dropdown-submenu-font-size;
-          color: $lang-dropdown-submenu-color;
```

Comment:
jest gdzieś indziej ta zmienna wykrozystyuwana ? jak nie, to tam na górze z deklaracji też można ją chyba wywalić jeżeli nie potrzebna ? Sprawdzisz?

Reply:
Nie jest wykorzystywana, już usunąłem.